src/services/produce_api/produceAPI/documentation.txt

```diff
@@ -0,0 +1,67 @@
+This is a simple file containing the documentation of this produce API developed in django and
```

Comment:
change this to a raml file.
Use this for guidance https://swagger.io/specification/